mscore/musescore.cpp

```diff
@@ -6018,6 +6018,33 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
                   cv->setMag(MagIdx::MAG_100, 1.0);
             setMag(1.0);
             }
+      // 2019.12.25: [Page-Width] as shortcut command
+      else if (cmd == "zoom-page-width") {
+            if (cv) {
+                // Store current zoom-level statically here for later retrieval
+                // instead of implementing it as a member variable since there's
+                // (currently) no other use for it.
+                MagIdx current_idx = MagIdx(mag->currentIndex());
+                static MagIdx previous_idx = current_idx;
+
+                // If current zoom-level is anything but page-width,
+                if (current_idx != MagIdx::MAG_PAGE_WIDTH) {
+                    // Save current zoom state
+                    previous_idx = current_idx;
+                    // Update zoom drop-down list
+                    mag->setMagIdx(MagIdx::MAG_PAGE_WIDTH);
+                    // Update the actual score's zoom-level
+                    mag->magChanged(MagIdx::MAG_PAGE_WIDTH);
+                }
```

Comment:
The right brace should also be indented.

> "As a base rule, place the left curly brace on the same line as the start of the statement and indent the closing one. This is know as Banner style indentation."

https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules